apps/web/src/app/api/ping/route.ts

```diff
@@ -2,6 +2,7 @@ import { NextResponse } from "next/server";
 
 export const dynamic = "force-dynamic";
 
-export async function GET() {
+export async function GET(req:Request) {
+  req.headers.set('OpenStatus-Ping',true)
```

Comment:
Hey @Mahmoudgalalz, this is the endpoint that we ping and want to know if it contains a `req.headers.get('OpenStatus-Ping') for example.

In here, we are calling all the endpoints to ping (e.g. `/api/ping`) and that's where we will need to add the header.
https://github.com/openstatusHQ/openstatus/blob/4242d8bd92c973e122b73a10a003464afb300adf/apps/web/src/app/api/checker/regions/_checker.ts#L82

Reply:
I have been looking inside the _checker all the time, but yeah I will do it rn and remove

so now there's no point form checking the header in the ping right ?!